src/core/Akka.TestKit/EventFilter/IEventFilterApplier.cs

```diff
@@ -97,7 +97,8 @@ public interface IEventFilterApplier
         /// </summary>
         /// <param name="expectedCount">The expected number of events</param>
         /// <param name="actionAsync">The async action.</param>
-        Task ExpectAsync(int expectedCount, Func<Task> actionAsync);
+        /// <param name="timeout"></param>
+        Task ExpectAsync(int expectedCount, Func<Task> actionAsync, TimeSpan? timeout = null);
```

Comment:
Breaking change here - need to add a method overload instead.

Reply:
Fixed